spec/devise_saml_authenticatable/saml_config_spec.rb

```diff
@@ -1,17 +1,9 @@
 require 'spec_helper'
 
 describe DeviseSamlAuthenticatable::SamlConfig do
-  subject(:saml_config) { controller.saml_config }
+  let(:saml_config) { controller.saml_config }
```

Comment:
We weren't sure if it would be cleaner to redefine `subject` on line 26, or just let-ize `saml_config` for the whole suite. It felt strange to keep subject when we end up changing it further on in the spec.